templates/flutter/pubspec.yaml.twig

```diff
@@ -22,7 +22,7 @@ dependencies:
   device_info_plus: ^9.0.2
   flutter_web_auth_2: ^3.0.0
   http: '>=0.13.6 <2.0.0'
-  package_info_plus: ^4.0.2
+  package_info_plus: ^5.0.1
```

Comment:
This is already handled in another PR.

Reply:
Downgraded, where's the PR which handle this change?